### ISSUE TITLE
No Stereo Audio + No True Mono (Left channel audio out only)

### ISSUE THREAD

**Issue by Dantasstic** [retrogamecorps/Simple30#2]
It appears that the only audio being output is the left channel audio. It is not a true mono combined track, as it is completely missing the right track information entirely.

Even with headphones plugged in, it simply plays the left side audio channel out of both speakers, instead of a left and right channel audio or a proper combined mono track from both ears.

EDIT: Here is a video I made regarding this issue, using beside a GBA for an example of a single speaker sound setup

https://youtu.be/L0o4sMfLWoE

**Comment by Grabou** [comment]
Is it the same behavior with the original S30 OS? This is a concern.

**Comment by Dantasstic** [comment]
Ok, I just popped in my stock micro sd card that came with the S30 and the audio issue is there as well.

Ugh, not good...

**Comment by Dantasstic** [comment]
@retrogamecorps Thoughts? 

IMHO, if this is a hardware design flaw, I will regret this purchase and would not buy this unit again. Having only half the audio available is a serious mark against the S30.

**Comment by retrogamecorps** [comment]
I haven't been able to find any sort of fix for this yet.  I worry that it is a hardware issue.  I could have sworn I mentioned this in my initial review video a couple months ago, but it's been so long I can't remember :)

**Comment by Dantasstic** [comment]
> 
> 
> I haven't been able to find any sort of fix for this yet. I worry that it is a hardware issue. I could have sworn I mentioned this in my initial review video a couple months ago, but it's been so long I can't remember :)

No, I just watched the video and you only mentioned that the speaker is tinny sounding (true) but no mention of it only outputting left channel audio. You mention it is a mono speaker a couple of times, and I would love it if it was actually mono, but it currently is not. 

If the game only has mono audio, you are good to go. If the game has stereo sound and no internal options to select mono instead of stereo, you are missing half the sound profile, which is not so good. :(

**Comment by retrogamecorps** [comment]
Ah, thanks for letting me know.  I think I was getting it confused with another device, I definitely remember testing one that only had one channel of audio.

**Comment by Grabou** [comment]
Some guy on Retro Game Handhelds Discord (#pocket-go-s30) made a DSP profile to force mono on RetroArch after I posted your video. I can't test since I haven't received yet my S30 but someone says it works. Maybe Russ should integrate that DSP in his next Simple release. ;) Thanks anyway for discovering this subtle problem!

**Comment by Dantasstic** [comment]
Yes, I think he responded to my other post! The Mono DSP profile does force true mono, which is nice! Here is the link:

[retroarch-audio-filters.zip](https://github.com/retrogamecorps/s30-add-on-pack/files/6094541/retroarch-audio-filters.zip)

I will link the comment chain here, just go all the way to the bottom for the relevant section haha

https://github.com/retrogamecorps/s30-add-on-pack/issues/12

**Comment by bkacjios** [comment]
I'm going to mess with getting alsamixer compiled for the S30 tomorrow, hopefully that will give me some insight on what the device is doing at a hardware level. I think the headphone jack supports stereo, but I'm not positive.

Hopefully, if it does indeed support stereo through headphones, I can edit the right config file and have it automatic switch between stereo and mono.

**Comment by Dantasstic** [comment]
> 
> 
> I'm going to mess with getting alsamixer compiled for the S30 tomorrow, hopefully that will give me some insight on what the device is doing at a hardware level. I think the headphone jack supports stereo, but I'm not positive.
> 
> Hopefully, if it does indeed support stereo through headphones, I can edit the right config file and have it automatic switch between stereo and mono.

I tried plugging headphones in, and it also only outputs the left channel audio to both ears (without the mono.dsp), same as it does to the single speaker. If you can fix that, that would be wonderful :)

**Comment by bkacjios** [comment]
That's a little worrisome. I have a feeling they just hooked up the left channel to both audio channels on the headphone jack.

**Comment by Dantasstic** [comment]
> 
> 
> That's a little worrisome. I have a feeling they just hooked up the left channel to both audio channels on the headphone jack.

Yes, that was my initial thought when I first posted about this. 

It _is_ made by PocketGo, and they are kind of known for having to fix hardware mistakes in their product runs; shame the first customers always seem to get the inferior products from them :/

**Comment by Decliniodell** [comment]
Hi everyone (I apologize for my English), I would like to know if anyone already has the answer to this problem, I bought my pocketgo s30 today and I was super disappointed that I didn't have stereo sound in the headset :(
